packages/amplication-data-service-generator-runner/src/main.ts

```diff
@@ -35,14 +34,17 @@ export default async function generateCode(
 ): Promise<void> {
   try {
     const resourceData = await readInputJson(source);
-    const { pluginInstallations } = resourceData;
-
-    await dynamicPackagesInstallations(pluginInstallations, defaultLogger);
 
     const modules = await createDataService(
       resourceData,
       defaultLogger,
-      join(__dirname, "..", AMPLICATION_MODULES)
+      join(
+        __dirname,
+        "..",
+        "..",
+        "..",
+        "libs/data-service-generator/node_modules"
+      )
```

Comment:
according to the code here - https://github.com/amplication/amplication/blob/2b00576375171b590a6f93744bac57e0aa43b57f/libs/data-service-generator/src/register-plugin.ts#L46

I think we can pass null